Client/Frontend/Browser/Search/InitialSearchEngines.swift

```diff
@@ -27,13 +27,8 @@ class InitialSearchEngines {
         }
         
         func excludedFromOnboarding(for locale: Locale) -> Bool {
-            switch self {
-                case .google, .bing, .duckduckgo, .yandex, .qwant, .startpage, .ecosia:
-                    return false
-                case .braveSearch:
-                    guard let region = locale.regionCode else { return true }
-                    return !InitialSearchEngines.braveSearchOnboardingRegions.contains(region)
-            }
+            // At the moment there are no custom rules to skip a SE in onboarding.
+            false
```

Comment:
May as well delete the whole function then? :)

Reply:
considered it but i had to remove a lot of test code etc.
i will remove it, we get rid of search engine onboarding soon anyway